src/scss/components/_form.scss

```diff
@@ -0,0 +1,85 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.form.css Form
+  *
+  * ## Form
+  * @demo demo/gaiden-css/components/form/demo.html
+**/
+
+.form {
+
+  &__container {
+    padding: {
+      right: 10%;
+      left: 10%;
+    }
+    width: 100%;
+  }
+
+  &__field {
+    display: block;
+
+    & + & {
+      margin-top: 16px;
+    }
+  }
+
+  &__input {
+    display: block;
+    padding: 13px;
+    border: 1px solid get-color((color: punch, opacity: .5));
+    border-radius: 6px;
```

Comment:
@GeorgeJegon we have a bunch of border-radius on all styleguide and designers still be undecided about which are the ideal border-radius to application, or if they will have more than one value to this. It is better wait this decision to make all this values a map or a variable.

*maybe use a `get-radius(input)` 😄